src/pages/certificates/create-certificate/create-certificate.js

```diff
@@ -78,11 +78,17 @@ const CreateCertificate = () => {
       }
     });
 
-    if (!isInvalid && check) {
-      setDisabled(false);
-    } else {
-      setDisabled(true);
-    }
+    const timerId = setTimeout(() => {
+      if (!isInvalid && check && email) {
+        setDisabled(false);
+      } else {
+        setDisabled(true);
+      }
+    }, [700]);
```

Comment:
What's that? ))

Reply:
Looks like both dependency array and latency. 
But what is it for? 
I meant to use regex

Reply:
I deleted this setTimeout. It isn't necessary here.

Reply:
I fixed.